--- mtk/package/boot/uboot-envtools/files/mediatek_filogic	2024-07-06 16:48:19.337447956 +0800
+++ imm/package/boot/uboot-envtools/files/mediatek_filogic	2024-07-06 16:14:34.967115369 +0800
@@ -39,7 +39,6 @@
 netcore,n60|\
 nokia,ea0326gmp|\
 qihoo,360t7|\
-qihoo,360t7-ubootmod|\
 tplink,tl-xdr4288|\
 tplink,tl-xdr6086|\
 tplink,tl-xdr6088|\
@@ -58,8 +57,6 @@
 bananapi,bpi-r4|\
 bananapi,bpi-r4-poe|\
 cmcc,rax3000m|\
-cmcc,rax3000m-nand-ubootmod|\
-cmcc,rax3000m-emmc-ubootmod|\
 jdcloud,re-cp-03)
 	. /lib/upgrade/common.sh
 
@@ -125,9 +122,7 @@
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x80000" "0x10000"
 	;;
 xiaomi,mi-router-ax3000t|\
-xiaomi,mi-router-wr30u-112m-nmbm|\
 xiaomi,mi-router-wr30u-stock|\
-xiaomi,redmi-router-ax6000|\
 xiaomi,redmi-router-ax6000-stock)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x10000" "0x20000"
 	ubootenv_add_uci_sys_config "/dev/mtd2" "0x0" "0x10000" "0x20000"
--- mtk/target/linux/mediatek/filogic/base-files/etc/board.d/01_leds	2024-07-06 16:49:45.157462036 +0800
+++ imm/target/linux/mediatek/filogic/base-files/etc/board.d/01_leds	2024-07-06 16:14:35.407115437 +0800
@@ -32,11 +32,6 @@
 	ucidef_set_led_netdev "lan3" "lan3" "blue:lan-3" "lan3" "link tx rx"
 	ucidef_set_led_netdev "wan" "wan" "blue:wan" "wan" "link tx rx"
 	;;
-cmcc,rax3000m|\
-cmcc,rax3000m-emmc-ubootmod|\
-cmcc,rax3000m-nand-ubootmod)
-	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1" "link"
-	;;
 cudy,re3000-v1)
 	ucidef_set_led_netdev "lan" "lan" "green:lan" "eth0" "link tx rx"
 	;;
@@ -119,12 +114,10 @@
 	ucidef_set_led_netdev "lan-2" "lan-2" "blue:lan-1" "lan2" "link tx rx"
 	ucidef_set_led_netdev "eth1" "lan-3" "blue:lan-2" "eth1" "link tx rx"
 	;;
-xiaomi,mi-router-wr30u-112m-nmbm|\
 xiaomi,mi-router-wr30u-stock|\
 xiaomi,mi-router-wr30u-ubootmod)
 	ucidef_set_led_netdev "wan" "wan" "blue:wan" "wan" "link tx rx"
 	;;
-xiaomi,redmi-router-ax6000|\
 xiaomi,redmi-router-ax6000-stock|\
 xiaomi,redmi-router-ax6000-ubootmod)
 	ucidef_set_led_netdev "wan" "wan" "rgb:network" "wan" "link"
--- mtk/target/linux/mediatek/filogic/base-files/etc/board.d/02_network	2024-07-06 16:50:42.097471369 +0800
+++ imm/target/linux/mediatek/filogic/base-files/etc/board.d/02_network	2024-07-06 16:14:35.407115437 +0800
@@ -13,8 +13,6 @@
 		;;
 	abt,asr3000|\
 	cmcc,rax3000m|\
-	cmcc,rax3000m-emmc-ubootmod|\
-	cmcc,rax3000m-nand-ubootmod|\
 	h3c,magic-nx30-pro|\
 	h3c,magic-nx30-pro-nmbm|\
 	imou,lc-hx3001-ubootmod|\
@@ -35,7 +33,6 @@
 	livinet,zr-3020|\
 	livinet,zr-3020-ubootmod|\
 	qihoo,360t7|\
-	qihoo,360t7-ubootmod|\
 	routerich,ax3000)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
 		;;
@@ -118,10 +115,8 @@
 		;;
 	xiaomi,mi-router-ax3000t|\
 	xiaomi,mi-router-ax3000t-ubootmod|\
-	xiaomi,mi-router-wr30u-112m-nmbm|\
 	xiaomi,mi-router-wr30u-stock|\
 	xiaomi,mi-router-wr30u-ubootmod|\
-	xiaomi,redmi-router-ax6000|\
 	xiaomi,redmi-router-ax6000-stock|\
 	xiaomi,redmi-router-ax6000-ubootmod)
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4" wan
@@ -187,10 +182,8 @@
 		;;
 	xiaomi,mi-router-ax3000t|\
 	xiaomi,mi-router-ax3000t-ubootmod|\
-	xiaomi,mi-router-wr30u-112m-nmbm|\
 	xiaomi,mi-router-wr30u-stock|\
 	xiaomi,mi-router-wr30u-ubootmod|\
-	xiaomi,redmi-router-ax6000|\
 	xiaomi,redmi-router-ax6000-stock|\
 	xiaomi,redmi-router-ax6000-ubootmod)
 		wan_mac=$(mtd_get_mac_ascii Bdata ethaddr_wan)
--- mtk/target/linux/mediatek/filogic/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac	2024-07-06 16:53:53.897503268 +0800
+++ imm/target/linux/mediatek/filogic/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac	2024-07-06 16:14:35.407115437 +0800
@@ -61,9 +61,7 @@
 	cmcc,a10-ubootmod)
 		[ "$PHYNBR" = "1" ] && mtd_get_mac_binary Factory 0xa > /sys${DEVPATH}/macaddress
 		;;
-	cmcc,rax3000m|\
-	cmcc,rax3000m-emmc-ubootmod|\
-	cmcc,rax3000m-nand-ubootmod)
+	cmcc,rax3000m)
 		addr=$(cat /sys/class/net/eth0/address)
 		[ "$PHYNBR" = "1" ] && macaddr_add $addr -1 > /sys${DEVPATH}/macaddress
 		;;
@@ -137,8 +135,7 @@
 	openembed,som7981)
 		[ "$PHYNBR" = "1" ] && cat /sys/class/net/eth0/address > /sys${DEVPATH}/macaddress
 		;;
-	qihoo,360t7|\
-	qihoo,360t7-ubootmod)
+	qihoo,360t7)
 		addr=$(mtd_get_mac_ascii factory lanMac)
 		[ "$PHYNBR" = "0" ] && macaddr_add $addr 2 > /sys${DEVPATH}/macaddress
 		[ "$PHYNBR" = "1" ] && macaddr_add $addr 3 > /sys${DEVPATH}/macaddress
--- mtk/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh	2024-07-06 16:56:30.837529000 +0800
+++ imm/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh	2024-07-06 16:14:35.407115437 +0800
@@ -99,8 +99,6 @@
 	bananapi,bpi-r4|\
 	bananapi,bpi-r4-poe|\
 	cmcc,rax3000m|\
-	cmcc,rax3000m-emmc-ubootmod|\
-	cmcc,rax3000m-nand-ubootmod|\
 	jdcloud,re-cp-03|\
 	mediatek,mt7988a-rfb|\
 	openwrt,one)
@@ -202,9 +200,7 @@
 	bananapi,bpi-r3|\
 	bananapi,bpi-r4|\
 	bananapi,bpi-r4-poe|\
-	cmcc,rax3000m|\
-	cmcc,rax3000m-emmc-ubootmod|\
-	cmcc,rax3000m-nand-ubootmod)
+	cmcc,rax3000m)
 		[ "$magic" != "d00dfeed" ] && {
 			echo "Invalid image type."
 			return 1
@@ -241,9 +237,7 @@
 	bananapi,bpi-r3-mini|\
 	bananapi,bpi-r4|\
 	bananapi,bpi-r4-poe|\
-	cmcc,rax3000m|\
-	cmcc,rax3000m-emmc-ubootmod|\
-	cmcc,rax3000m-nand-ubootmod)
+	cmcc,rax3000m)
 		case "$(fitblk_get_bootdev)" in
 		mmcblk*)
 			emmc_copy_config
--- mtk/target/linux/mediatek/image/filogic.mk	2024-07-06 16:57:11.997535686 +0800
+++ imm/target/linux/mediatek/image/filogic.mk	2024-07-06 16:14:35.407115437 +0800
@@ -516,44 +516,6 @@
 endef
 TARGET_DEVICES += cmcc_rax3000m
 
-define Device/cmcc_rax3000m-emmc-ubootmod
-  DEVICE_VENDOR := CMCC
-  DEVICE_MODEL := RAX3000M eMMC version
-  DEVICE_VARIANT := (Custom U-Boot layout)
-  DEVICE_DTS := mt7981b-cmcc-rax3000m-emmc-ubootmod
-  DEVICE_DTS_DIR := ../dts
-  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware kmod-usb3 \
-	automount f2fsck mkf2fs
-  KERNEL := kernel-bin | lzma | fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
-  KERNEL_INITRAMFS := kernel-bin | lzma | \
-	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb with-initrd | pad-to 64k
-  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
-endef
-TARGET_DEVICES += cmcc_rax3000m-emmc-ubootmod
-
-define Device/cmcc_rax3000m-nand-ubootmod
-  DEVICE_VENDOR := CMCC
-  DEVICE_MODEL := RAX3000M NAND version
-  DEVICE_VARIANT := (Custom U-Boot layout)
-  DEVICE_DTS := mt7981b-cmcc-rax3000m-nand-ubootmod
-  DEVICE_DTS_DIR := ../dts
-  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware kmod-usb3 \
-	automount f2fsck mkf2fs
-  UBINIZE_OPTS := -E 5
-  BLOCKSIZE := 128k
-  PAGESIZE := 2048
-  IMAGE_SIZE := 116736k
-  KERNEL_IN_UBI := 1
-  IMAGES += factory.bin
-  IMAGE/factory.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
-  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
-  KERNEL = kernel-bin | lzma | \
-	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
-  KERNEL_INITRAMFS = kernel-bin | lzma | \
-	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb with-initrd
-endef
-TARGET_DEVICES += cmcc_rax3000m-nand-ubootmod
-
 define Device/comfast_cf-e393ax
   DEVICE_VENDOR := Comfast
   DEVICE_MODEL := CF-E393AX
@@ -1264,29 +1226,6 @@
 endef
 TARGET_DEVICES += qihoo_360t7
 
-define Device/qihoo_360t7-ubootmod
-  DEVICE_VENDOR := Qihoo
-  DEVICE_MODEL := 360T7
-  DEVICE_VARIANT := (Custom U-Boot layout)
-  DEVICE_DTS := mt7981b-qihoo-360t7-ubootmod
-  DEVICE_DTS_DIR := ../dts
-  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware
-  SUPPORTED_DEVICES += qihoo,360-t7-ubootmod
-  UBINIZE_OPTS := -E 5
-  BLOCKSIZE := 128k
-  PAGESIZE := 2048
-  IMAGE_SIZE := 65536k
-  KERNEL_IN_UBI := 1
-  IMAGES += factory.bin
-  IMAGE/factory.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
-  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
-  KERNEL = kernel-bin | lzma | \
-	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
-  KERNEL_INITRAMFS = kernel-bin | lzma | \
-	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb with-initrd
-endef
-TARGET_DEVICES += qihoo_360t7-ubootmod
-
 define Device/routerich_ax3000
   DEVICE_VENDOR := Routerich
   DEVICE_MODEL := AX3000
@@ -1466,28 +1405,6 @@
 endef
 TARGET_DEVICES += xiaomi_mi-router-ax3000t-ubootmod
 
-define Device/xiaomi_mi-router-wr30u-112m-nmbm
-  DEVICE_VENDOR := Xiaomi
-  DEVICE_MODEL := Mi Router WR30U
-  DEVICE_VARIANT := (Custom U-Boot layout)
-  DEVICE_DTS := mt7981b-xiaomi-mi-router-wr30u-112m-nmbm
-  DEVICE_DTS_DIR := ../dts
-  UBINIZE_OPTS := -E 5
-  BLOCKSIZE := 128k
-  PAGESIZE := 2048
-  IMAGE_SIZE := 114688k
-  KERNEL_IN_UBI := 1
-  DEVICE_PACKAGES := kmod-mt7915e kmod-mt7981-firmware mt7981-wo-firmware
-ifneq ($(CONFIG_TARGET_ROOTFS_INITRAMFS),)
-  ARTIFACTS := initramfs-factory.ubi
-  ARTIFACT/initramfs-factory.ubi := append-image-stage initramfs-kernel.bin | ubinize-kernel
-endif
-  IMAGES += factory.bin
-  IMAGE/factory.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
-  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
-endef
-TARGET_DEVICES += xiaomi_mi-router-wr30u-112m-nmbm
-
 define Device/xiaomi_mi-router-wr30u-stock
   DEVICE_VENDOR := Xiaomi
   DEVICE_MODEL := Mi Router WR30U
@@ -1535,25 +1452,6 @@
 endef
 TARGET_DEVICES += xiaomi_mi-router-wr30u-ubootmod
 
-define Device/xiaomi_redmi-router-ax6000
-  DEVICE_VENDOR := Xiaomi
-  DEVICE_MODEL := Redmi Router AX6000
-  DEVICE_VARIANT := (Custom U-Boot layout)
-  DEVICE_DTS := mt7986a-xiaomi-redmi-router-ax6000
-  DEVICE_DTS_DIR := ../dts
-  DEVICE_PACKAGES := kmod-leds-ws2812b kmod-mt7915e kmod-mt7986-firmware mt7986-wo-firmware
-  KERNEL_LOADADDR := 0x48000000
-  UBINIZE_OPTS := -E 5
-  BLOCKSIZE := 128k
-  PAGESIZE := 2048
-  IMAGE_SIZE := 112640k
-  KERNEL_IN_UBI := 1
-  IMAGES += factory.bin
-  IMAGE/factory.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
-  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
-endef
-TARGET_DEVICES += xiaomi_redmi-router-ax6000
-
 define Device/xiaomi_redmi-router-ax6000-stock
   DEVICE_VENDOR := Xiaomi
   DEVICE_MODEL := Redmi Router AX6000
