--- qcam/package/boot/uboot-envtools/files/qualcommax	2024-06-13 23:07:49.344866096 +0800
+++ lede/package/boot/uboot-envtools/files/qualcommax	2024-06-16 09:34:52.576166536 +0800
@@ -48,7 +48,6 @@
 	[ -n "$idx" ] && \
 		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x20000" "0x20000" "1"
 	;;
-jdc,ax1800-pro|\
 redmi,ax5-jdcloud)
 	mmcpart="$(find_mmc_part 0:APPSBLENV)"
 	[ -n "$mmcpart" ] && \
--- qcam/package/firmware/ipq-wifi/Makefile	2024-06-13 23:07:49.344866096 +0800
+++ lede/package/firmware/ipq-wifi/Makefile	2024-06-16 09:34:52.596166535 +0800
@@ -36,7 +36,6 @@
 	edimax_cax1800 \
 	glinet_gl-ax1800 \
 	glinet_gl-axt1800 \
-	jdc_ax1800-pro \
 	linksys_mr7350 \
 	linksys_mx4200 \
 	linksys_mx5300 \
@@ -144,7 +143,6 @@
 $(eval $(call generate-ipq-wifi-package,edimax_cax1800,Edimax CAX1800))
 $(eval $(call generate-ipq-wifi-package,glinet_gl-ax1800,GL.iNet GL-AX1800))
 $(eval $(call generate-ipq-wifi-package,glinet_gl-axt1800,GL.iNet GL-AXT1800))
-$(eval $(call generate-ipq-wifi-package,jdc_ax1800-pro,JD Cloud AX1800 Pro))
 $(eval $(call generate-ipq-wifi-package,linksys_mr7350,Linksys MR7350))
 $(eval $(call generate-ipq-wifi-package,linksys_mx4200,Linksys MX4200))
 $(eval $(call generate-ipq-wifi-package,linksys_mx5300,Linksys MX5300))
--- qcam/target/linux/qualcommax/image/ipq60xx.mk	2024-06-14 18:09:57.657927693 +0800
+++ lede/target/linux/qualcommax/image/ipq60xx.mk	2024-06-16 09:34:53.496166193 +0800
@@ -61,20 +61,6 @@
 endef
 TARGET_DEVICES += glinet_gl-axt1800
 
-define Device/jdc_ax1800-pro
-	$(call Device/FitImage)
-	$(call Device/EmmcImage)
-	DEVICE_VENDOR := JD Cloud
-	DEVICE_MODEL := JDC AX1800 Pro
-	SOC := ipq6000
-	KERNEL_SIZE := 6144k
-	BLOCKSIZE := 64k
-	DEVICE_DTS_CONFIG := config@cp03-c2
-	IMAGE/factory.bin := append-kernel | pad-to $$$$(KERNEL_SIZE) |  append-rootfs | append-metadata
-	DEVICE_PACKAGES := ipq-wifi-jdc_ax1800-pro kmod-fs-ext4 mkf2fs f2fsck kmod-fs-f2fs
-endef
-TARGET_DEVICES += jdc_ax1800-pro
-
 define Device/linksys_mr7350
 	$(call Device/FitImage)
 	DEVICE_VENDOR := Linksys
--- qcam/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/01_leds	2024-06-13 23:26:55.595054861 +0800
+++ lede/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/01_leds	2024-06-16 09:34:53.496166193 +0800
@@ -11,9 +11,6 @@
 	ucidef_set_led_netdev "wlan2g" "WLAN2G" "blue:wlan2g" "wlan1"
 	ucidef_set_led_netdev "wlan5g" "WLAN5G" "blue:wlan5g" "wlan0"
 	;;
-jdc,ax1800-pro)
-	ucidef_set_led_netdev "wan" "WAN" "net_blue" "wan"
-	;;
 redmi,ax5-*|\
 xiaomi,rm1800)
 	ucidef_set_led_netdev "internet" "Internet" "blue:network" "wan"
--- qcam/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network	2024-06-13 23:07:49.344866096 +0800
+++ lede/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network	2024-06-16 09:34:53.496166193 +0800
@@ -12,7 +12,6 @@
 
 	case "$board" in
 	cmiot,ax18|\
-	jdc,ax1800-pro|\
 	qihoo,360v6|\
 	redmi,ax5-*|\
 	xiaomi,rm1800)
--- qcam/target/linux/qualcommax/ipq60xx/base-files/etc/hotplug.d/firmware/11-ath11k-caldata	2024-06-16 09:39:40.276058787 +0800
+++ lede/target/linux/qualcommax/ipq60xx/base-files/etc/hotplug.d/firmware/11-ath11k-caldata	2024-06-16 09:34:53.496166193 +0800
@@ -16,7 +16,6 @@
 	xiaomi,rm1800)
 		caldata_extract "0:art" 0x1000 0x10000
 		;;
-	jdc,ax1800-pro|\
 	redmi,ax5-jdcloud)
 		caldata_extract_mmc "0:ART" 0x1000 0x10000
 		;;
--- qcam/target/linux/qualcommax/ipq60xx/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac	2024-06-13 23:07:49.344866096 +0800
+++ lede/target/linux/qualcommax/ipq60xx/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac	2024-06-16 09:34:53.496166193 +0800
@@ -15,10 +15,6 @@
 		[ "$PHYNBR" = "0" ] && macaddr_add $(get_mac_label) 3 > /sys${DEVPATH}/macaddress
 		[ "$PHYNBR" = "1" ] && macaddr_add $(get_mac_label) 2 > /sys${DEVPATH}/macaddress
 		;;
-	jdc,ax1800-pro)
-		[ "$PHYNBR" = "0" ] && macaddr_add $(get_mac_label) 2 > /sys${DEVPATH}/macaddress
-		[ "$PHYNBR" = "1" ] && macaddr_add $(get_mac_label) 1 > /sys${DEVPATH}/macaddress
-		;;
 	linksys,mr7350)
 		addr=$(mtd_get_mac_ascii devinfo hw_mac_addr)
 		[ "$PHYNBR" = "0" ] && macaddr_add $addr 2 > /sys${DEVPATH}/macaddress
--- qcam/target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh	2024-06-14 18:15:48.697986594 +0800
+++ lede/target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh	2024-06-16 09:34:53.496166193 +0800
@@ -31,7 +31,6 @@
 		fw_setenv auto_recovery yes
 		nand_do_upgrade "$1"
 		;;
-	jdc,ax1800-pro|\
 	redmi,ax5-jdcloud)
 		kernelname="0:HLOS"
 		rootfsname="rootfs"
