--- ./rax/11_fix_wifi_mac	2024-01-07 10:28:57.024766000 +0800
+++ ./imm/11_fix_wifi_mac	2024-01-10 16:47:10.085797700 +0800
@@ -58,8 +58,7 @@
 		[ "$PHYNBR" = "0" ] && macaddr_add $addr 1 > /sys${DEVPATH}/macaddress
 		[ "$PHYNBR" = "1" ] && macaddr_add $addr 2 > /sys${DEVPATH}/macaddress
 		;;
-	cmcc,a10-ubootmod|\
-	cmcc,rax3000m-nand-ubootmod)
+	cmcc,a10-ubootmod)
 		[ "$PHYNBR" = "1" ] && mtd_get_mac_binary Factory 0xa > /sys${DEVPATH}/macaddress
 		;;
 	cmcc,rax3000m)
@@ -73,9 +72,6 @@
 		esac
 		[ "$PHYNBR" = "1" ] && echo "$addr" > /sys${DEVPATH}/macaddress
 		;;
-	cmcc,rax3000m-emmc-ubootmod)
-		[ "$PHYNBR" = "1" ] && mmc_get_mac_binary factory 0xa > /sys${DEVPATH}/macaddress
-		;;
 	cudy,wr3000-v1)
 		addr=$(mtd_get_mac_binary bdinfo 0xde00)
 		# Originally, phy0 is phy1 mac with LA bit set. However, this would conflict
