--- ./rax/mediatek_filogic	2024-01-10 15:24:21.053898700 +0800
+++ ./imm/mediatek_filogic	2024-01-10 16:47:09.495797700 +0800
@@ -43,7 +43,6 @@
 	;;
 cetron,ct3003-stock|\
 cetron,ct3003-ubootmod|\
-cmcc,rax3000m-nand-ubootmod|\
 h3c,magic-nx30-pro-nmbm|\
 netgear,wax220|\
 zbtlink,zbt-z8102ax|\
@@ -67,9 +66,6 @@
 		;;
 	esac
 	;;
-cmcc,rax3000m-emmc-ubootmod)
-	ubootenv_add_uci_config "/dev/mmcblk0p1" "0x0" "0x80000" "0x80000"
-	;;
 glinet,gl-mt2500|\
 glinet,gl-mt6000)
 	local envdev=$(find_mmc_part "u-boot-env")
