--- ./rax/platform.sh	2024-01-10 15:25:46.225395400 +0800
+++ ./imm/platform.sh	2024-01-10 16:47:10.085797700 +0800
@@ -116,7 +116,6 @@
 	yuncore,ax835)
 		default_do_upgrade "$1"
 		;;
-	cmcc,rax3000m-emmc-ubootmod|\
 	glinet,gl-mt2500|\
 	glinet,gl-mt6000)
 		CI_KERNPART="kernel"
@@ -197,7 +196,6 @@
 platform_copy_config() {
 	case "$(board_name)" in
 	acer,predator-w6|\
-	cmcc,rax3000m-emmc-ubootmod|\
 	glinet,gl-mt2500|\
 	glinet,gl-mt6000|\
 	jdcloud,re-cp-03|\
